src/chart.zoom.js

```diff
@@ -527,10 +527,16 @@ var zoomPlugin = {
 			var endX = Math.max(beginPoint.clientX, endPoint.clientX) - offsetX;
 			var rectWidth = endX - startX;
 
+			var dragOptions = chartInstance.options.zoom.drag;
 
-			ctx.fillStyle = 'rgba(225,225,225,0.3)';
-			ctx.lineWidth = 5;
+			ctx.fillStyle = dragOptions.backgroundColor || 'rgba(225,225,225,0.3)';
 			ctx.fillRect(startX, yAxis.top, rectWidth, yAxis.bottom - yAxis.top);
+
+			if (dragOptions.borderWidth > 0) {
```

Comment:
We should keep it working when `borderWidth` is `undefined`. You could change these two lines to:
```
if (dragOptions.borderWidth !== 0) {
	ctx.lineWidth = dragOptions.borderWidth || 5;
```

Reply:
If we do this, we'll introduce a change since there is no border currently. Are you OK with this?

Reply:
Ah, you're right. I saw we had `ctx.lineWidth = 5` before, but I guess that wasn't used